db/seeds.rb

```diff
@@ -5,3 +5,85 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# I have added some objects whose status is Inactive to ensure they don't appear.
+# Need to create
+# A user for login
+# A manager role, and assign it to the admin user
+# Other roles
+# Departments
+# People
+
+manager_role = Role.find_or_create_by(name: "Manager")
+if User.count == 0
+  password = (0...6).map { ('a'..'z').to_a[rand(26)] }.join.upcase
+  puts "The initial user password is #{password}"
+  puts "The initial user name is 'bdoe@example.com'"
+  admin_user = User.create(username: "bdoe",
+                           email: "bdoe@example.com",
+                           firstname: "Bob", lastname: "Doe",
+                           password: password,
+                           password_confirmation: password
+                           )
+  admin_user.roles << manager_role
+end
+
+Role.find_or_create_by(name: "Editor")
+Role.find_or_create_by(name: "Author")
+Role.find_or_create_by(name: "Depositor")
+Role.find_or_create_by(name: "Trainer")
+Role.find_or_create_by(name: "Admin")
+
+# Departments are where people call home.
+# I need another status to determine visbility on the various drop downs.
```

Comment:
"visibility" (spelling)

But also, I don't understand this comment.  Is this a TODO?